core/templates/dev/head/base.html

```diff
@@ -278,8 +278,9 @@
       {{ include_js_file('services/activeInputData.js') }}
       {{ include_js_file('services/warningsData.js') }}
 
-      {{ include_js_file('components/ExplorationSummaryTileDirective.js') }}
       {{ include_js_file('components/ObjectEditorDirective.js') }}
+      {{ include_js_file('services/explorationContextService.js') }}
```

Comment:
Consider following the existing pattern, and grouping services/... with the block above this one.